### PR TITLE
Task 5 University Group

### PR DESCRIPTION
**Pull request by IrynaSemenenko**
Class main is displayed all information about groups: name, years of study and students.
Condition: if students were not added, then the program updates the list of students.

### CODE REVIEW

File: ClassMethod/src/UniversityGroup.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        this.studentArray = studentArray;
+        setEndYear();
+    }
+    public String[] addStudents() {
+        String[] students;
+        students = new String[]{"Nikita", "Anatoliy", "Olga"};
+        return students;
+    }
+    public void getGroupDescription() {
+        if (getStudentArray() == null) {
```
Comment (Hmeliumm) [inline ClassMethod/src/UniversityGroup.java:40]: Please make this check if addStudents

File: ClassMethod/src/UniversityGroup.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    }
+    public UniversityGroup(String groupName, int startYear, String[] studentArray) {
+        this.groupName = groupName;
+        this.startYear = startYear;
+        this.studentArray = studentArray;
+        setEndYear();
+    }
+    public String[] addStudents() {
+        String[] students;
+        students = new String[]{"Nikita", "Anatoliy", "Olga"};
```
Comment (Hmeliumm) [inline ClassMethod/src/UniversityGroup.java:36]: Please add students in main, not as hard-code

File: ClassMethod/src/UniversityGroup.java [hunk 3]
```diff
@@ -0,0 +1,51 @@
+import java.util.Arrays;
+
+public class UniversityGroup {
+    private final String groupName;
+    private final int startYear;
+    private int endYear;
+    private String[] studentArray;
+    public String getGroupName() {
```
Comment (Hmeliumm) [inline ClassMethod/src/UniversityGroup.java:8]: Please delete as it unused

File: ClassMethod/src/UniversityGroup.java [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+        return groupName;
+    }
+    public int getStartYear() {
```
Comment (Hmeliumm) [inline ClassMethod/src/UniversityGroup.java:11]: Please delete as it unused

File: ClassMethod/src/UniversityGroup.java [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+        return startYear;
+    }
+    public String[] getStudentArray() {
+        return studentArray;
+    }
+    public int getEndYear() {
```
Comment (Hmeliumm) [inline ClassMethod/src/UniversityGroup.java:17]: Please delete as it unused

File: ClassMethod/src/UniversityGroup.java [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+        return endYear;
+    }
+    public int setEndYear() {
+        return startYear + 5;
```
Comment (Hmeliumm) [inline ClassMethod/src/UniversityGroup.java:21]: This should be actually set of endYearField

File: ClassMethod/src/GroupsInfo.java [hunk 7]
```diff
@@ -0,0 +1,10 @@
+public class GroupsInfo {
+
+    public static void main(String[] args) {
+        UniversityGroup firstGroupStudent = new UniversityGroup("FirstGroup", 2020, new String[]{"Anna", "Ira", "Anton"});
+        firstGroupStudent.getGroupDescription();
+        System.out.println();
```
Comment (Hmeliumm) [inline ClassMethod/src/GroupsInfo.java:6]: Please delete

**Review by Hmeliumm — CHANGES_REQUESTED** [review]

File: ClassMethod/src/UniversityGroup.java (hunk 1 above)
Comment (IrynaSemenenko) [reply ClassMethod/src/UniversityGroup.java]: Done

File: ClassMethod/src/UniversityGroup.java (hunk 2 above)
Comment (IrynaSemenenko) [reply ClassMethod/src/UniversityGroup.java]: Done

File: ClassMethod/src/UniversityGroup.java (hunk 3 above)
Comment (IrynaSemenenko) [reply ClassMethod/src/UniversityGroup.java]: Done

File: ClassMethod/src/UniversityGroup.java (hunk 4 above)
Comment (IrynaSemenenko) [reply ClassMethod/src/UniversityGroup.java]: Done

File: ClassMethod/src/UniversityGroup.java (hunk 5 above)
Comment (IrynaSemenenko) [reply ClassMethod/src/UniversityGroup.java]: Done

File: ClassMethod/src/UniversityGroup.java (hunk 6 above)
Comment (IrynaSemenenko) [reply ClassMethod/src/UniversityGroup.java]: I removed it

**Review by Hmeliumm — APPROVED** [review]

**Pull request merged**